packages/auth-construct/README.md

```diff
@@ -0,0 +1,133 @@
+# AmplifyAuth Construct
+
+This package vends an L3 CDK Construct that enables faster, easier and secure app authentication and authorization powered by Amazon Cognito. Amplify Auth lets you quickly set up secure authentication flows with a fully-managed user directory. Control what users have access to in your mobile and web apps with Amplify Auth's built-in authorization capabilities.
```

Comment:
nit: I always forget what "L3 CDK Construct" is. Would be helpful if we could add a link https://docs.aws.amazon.com/cdk/v2/guide/constructs.html

Reply:
im not sure about this one since data team did not include links to other cdk docs https://constructs.dev/packages/@aws-amplify/data-construct/v/1.4.2?lang=typescript